docs/en/setup/backend/exporter.md

```diff
@@ -0,0 +1,161 @@
+# Exporter
+SkyWalking provides the essential functions of metrics aggregation, trace, log, alarm, and analysis.
+In many real-world scenarios, users may want to forward their data to a 3rd party system for further in-depth analysis.
+**Exporter** has made that possible.
+
+The exporter is an independent module that has to be manually activated.
+
+Right now, we provide the following exporters:
+1. gRPC Exporter
+- Metrics
+2. Kafka Exporter
+- Trace
+- Log
+
+## gRPC Exporter
+### Metrics gRPC Exporter
+Metrics gRPC exporter uses SkyWalking's native exporter service definition. Here is the proto definition.
+```proto
+service MetricExportService {
+    rpc export (stream ExportMetricValue) returns (ExportResponse) {
+    }
+
+    rpc subscription (SubscriptionReq) returns (SubscriptionsResp) {
+    }
+}
+
+message ExportMetricValue {
+    string metricName = 1;
+    string entityName = 2;
+    string entityId = 3;
+    ValueType type = 4;
+    int64 timeBucket = 5;
+    int64 longValue = 6;
+    double doubleValue = 7;
+    repeated int64 longValues = 8;
+}
+
+message SubscriptionsResp {
+    repeated SubscriptionMetric metrics = 1;
+}
+
+message SubscriptionMetric {
+    string metricName = 1;
+    EventType eventType = 2;
+}
+
+enum ValueType {
+    LONG = 0;
+    DOUBLE = 1;
+    MULTI_LONG = 2;
+}
+
+enum EventType {
+    // The metrics aggregated in this bulk, not include the existing persistent data.
+    INCREMENT = 0;
+    // Final result of the metrics at this moment.
+    TOTAL = 1;
+}
+
+message SubscriptionReq {
+
+}
+
+message ExportResponse {
+}
+```
+
+To activate the exporter, you should add this into your `application.yml`
```

Comment:
I believe this is copied from the previous doc. We should adjust words. These configurations exist in the YAML file already, we should say `you should set xxx to true to activate yyy`.

Reply:
Same suggestions for trace and log exporter configurations.